### DISCUSSION TITLE
Bug? Blank page with commenting lines

### DISCUSSION THREAD

**Discussion by constantcopyandwork** [thuliteio/doks#760] [Q&A]
Does anyone know why commenting lines in the blog single.html page generate blank pages?

I was working on a new section to add to the layout but wasn't done, so I commented it out. Suddenly all my blog pages went blank.

Thanks!

**Comment by h-enk** [comment]
It's a Hugo thing — see https://github.com/gohugoio/hugo/issues/7243#issuecomment-624029653
